Userland/Libraries/LibCore/DateTime.cpp

```diff
@@ -251,8 +251,10 @@ String DateTime::to_string(StringView format) const
                 format_time_zone_offset(true);
                 break;
             case 'Z': {
-#ifndef __FreeBSD__
+#if !defined(__FreeBSD__) && !defined(AK_OS_WIN32)
                 auto const* timezone_name = tzname[daylight];
+#elif defined(AK_OS_WIN32)
+                auto const* timezone_name = "";
 #else
                 auto const* timezone_name = tzname[0];
```

Comment:
Maybe I'm the only one, but I fond the double negatives to be a bit weird and confusing. Wouldn't it make more sense to simply use:
```
#if defined(AK_OS_WIN32)
...
#elif defined(__FreeBSD__)
...
#else
```